src/krylov/conv_params.h

```diff
@@ -0,0 +1,49 @@
+/******************************************************************************
+ * Copyright (c) 1998 Lawrence Livermore National Security, LLC and other
+ * HYPRE Project Developers. See the top-level COPYRIGHT file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ ******************************************************************************/
+
+/******************************************************************************
+ *
+ * Krylov convergence parameters header
+ *
+ *****************************************************************************/
+
+#ifndef hypre_CONV_PARAMS_HEADER
+#define hypre_CONV_PARAMS_HEADER
+
+
+/**
+ * The {\tt hypre\_conv\_params} object ...
+ **/
+
+/*
+ Summary of Parameters to Control Stopping Test:
+ - tol = relative error tolerance, as above
+ -a_tol = absolute convergence tolerance (default is 0.0)
+   If one desires the convergence test to check the absolute
+   convergence tolerance *only*, then set the relative convergence
+   tolerance to 0.0.  (The default convergence test is  <C*r,r> <=
+   max(relative_tolerance^2 * <C*b, b>, absolute_tolerance^2)
+- cf_tol = convergence factor tolerance; if >0 used for special test
+  for slow convergence
+ - atolf = absolute error tolerance factor to be used _together_ with the
+ relative error tolerance, |delta-residual| / ( atolf + |right-hand-side| ) < tol
+  (To BE PHASED OUT)
+*/
+
+typedef struct
+{
+   HYPRE_Real   tol;
+   HYPRE_Real   atolf;
+   HYPRE_Real   cf_tol;
+   HYPRE_Real   a_tol;
+   HYPRE_Real   rtol;
+
+   HYPRE_Real   rel_residual_norm;
+
+} hypre_conv_params;
```

Comment:
This doesn't conform to our naming conventions.  Should be `hypre_ConvParams`.

Reply:
@rfalgout yes, I agree. I expected to change this at some point.